tools/accuracy_checker/accuracy_checker/metrics/im2latex_images_match.py

```diff
@@ -0,0 +1,358 @@
+"""
+Copyright (c) 2020 Intel Corporation
```

Comment:
Please align copyright
```suggestion
Copyright (c) 2018-2020 Intel Corporation
```